README.md

```diff
@@ -7,39 +7,45 @@
 
 ## Screenshot
 
+![Screenshot from 2022-03-16 15-10-02](https://user-images.githubusercontent.com/88809610/158707611-ab751c86-6740-4f92-99d3-40ac86214c08.png)
 
 ## Built With
 
 - HTML, CSS, JavaScript
 - Technologies used: GitHub, VScode, nodejs, React, GIT.
 
 
-
-## How it works
-
-
-
-
 ## Live Demo
 
-[Live Demo Link]()
-
+Click [here](https://forhemer.github.io/Math-Magicians/) to see the live demo.
 
```

Comment:
- [ ] (Optional) The _**Live Demo Link**_ does not show the project, I would suggest you either deploy it properly or remove it from the `README.md` file. 👍